cmd/config.js

```diff
@@ -20,6 +20,7 @@ const logger = require('../lib/logger.js');
 const project = require('../lib/project.js');
 const user = require('../lib/user.js');
 const util = require('../lib/util.js');
+const handleActionFailure = require('./../lib/util').handleActionFailure;
```

Comment:
Nit: I would rename `handleActionFailure` to `handleCommandFailure` since it's more specific